ccdproc/image_collection.py

```diff
@@ -216,6 +222,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str or int, The extension from which the header and data will be read in all files.
```

Comment:
Could be split into two lines (because > 80 characters). Also the "in all files" part that I mentioned in the parameter description.

And did you check if one can pass in a tuple of `(str, int)`?